ietfdata/datatracker.py

```diff
@@ -673,8 +683,10 @@ def person_from_email(self, email_addr: str) -> Optional[Person]:
             return None
 
 
-    def person_aliases(self, person: Person) -> Iterator[PersonAlias]:
-        url = "/api/v1/person/alias/?person=" + str(person.id)
+    def person_aliases(self, person: Optional[Person] = None) -> Iterator[PersonAlias]:
+        url = "/api/v1/person/alias/"
+        if person is not None:
+            url += "?person=" + str(person.id)
```

Comment:
Would it be better to leave `person_aliases()` unchanged, and add `people_aliases(self, since, until)` as well?

Reply:
The /person/alias endpoint doesn't have timestamps (or allow filtering by timestamp), unfortunately. 

Originally, my thought was to fetch aliases independently of person objects, but without being able to filter by timestamp, this isn't practical (you'd have to fetch them all every time). So, I don't think I need this change anymore, but I'm unsure if it's worth including for other uses.

Reply:
I'd leave it out if we can't filter them. Any change to a person that creates a new alias will presumably show up as an update to the Person object anyway, so it's possible to tell when to re-fetch the aliases for a person?

Reply:
Agreed; rolled this change back.

I presume that a new alias would result in the person object's timestamp being updated, but I'd need to experiment to check that.